public/locales/en-US/translations.json

```diff
@@ -380,12 +380,17 @@
   "buy_offers":"Buy Offers",
   "sell_offers":"Sell Offers",
   "offer_index": "Offer ID",
-  "owner": "Owner's Account",
```

Comment:
Does removing this negatively impact any other pages?

Reply:
It will not.  I found no usages for it.

Reply:
It is used by my other PR